### PR TITLE
Bugfix/data types 2

### PR DESCRIPTION
**Pull request by mreiche**
Fixes the bug, when you want to perform a pandas operation on an empty list, which is returned when no data is present:
```
ERROR:yfinance:XXX: No timezone found, symbol may be delisted
ERROR:xxx:'list' object has no attribute 'loc'
```
This change does 2 things:
- It returns an static empty pandas series instead of a list
- It annotates the return type for some functions

### CODE REVIEW

**Comment by mreiche** [comment]
I've updated the description. Please review.

File: yfinance/base.py [hunk 1]
```diff
@@ -45,6 +45,9 @@
 from .const import _BASE_URL_, _ROOT_URL_
 
 
+_empty_series = pd.Series()
```
Comment (ValueRaider) [inline yfinance/base.py:48]: Why?
- cost of `pd.Series()` is negligible
- `_empty_series` is more characters

File: requirements.txt [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
 multitasking>=0.0.7
 lxml>=4.9.1
 appdirs>=1.4.4
 pytz>=2022.5
 frozendict>=2.3.4
 beautifulsoup4>=4.11.1
 html5lib>=1.1
-peewee>=3.16.2
\ No newline at end of file
+peewee>=3.16.2
```
Comment (ValueRaider) [inline requirements.txt:13]: revert changes

File: tests/ticker.py [hunk 3]
```diff
@@ -798,7 +814,7 @@ def test_info(self):
 def suite():
     suite = unittest.TestSuite()
     suite.addTest(TestTicker('Test ticker'))
-    suite.addTest(TestTickerEarnings('Test earnings'))
+    #suite.addTest(TestTickerEarnings('Test earnings'))
```
Comment (ValueRaider) [inline tests/ticker.py:817]: why comment?

**Review by ValueRaider — CHANGES_REQUESTED** [review]
Nit-picky comments, but IMO every code change should be relevant and matter.

File: requirements.txt (hunk 2 above)
Comment (mreiche) [reply requirements.txt]: I reverted the explicit dependencies but kept the newline at the end.

File: tests/ticker.py (hunk 3 above)
Comment (mreiche) [reply tests/ticker.py]: I branched from the wrong base I guess. Reverted.

**Comment by ValueRaider** [comment]
You might have a better time with Git if you rebase your changes.

**Comment by mreiche** [comment]
> You might have a better time with Git if you rebase your changes.

I don't get what happend acutally. I rebased from master to dev and git messed something up. This is the second branch already. I usually don't use rebase because of these issues.

Anyway, is there something you request a change for?

**Comment by mreiche** [comment]
> I'll merge this time, but in future:
> 
> * reference [Network Graph](https://github.com/ranaroussi/yfinance/network), keep it tidy
> * avoid merging in from both `main` and `dev`, it usually creates a mess

Alright

**Review by ValueRaider — APPROVED** [review]

**Pull request merged**